src/actions.ts

```diff
@@ -148,6 +148,12 @@ export default class ActionCreator extends BaseActionCreator {
     return this.fetchJSON<LibraryData>(ActionCreator.GET_ONE_LIBRARY, url).bind(this);
   }
 
+  search(data: FormData) {
+    let url = "/admin/libraries/search_details";
+    let result = this.postForm(ActionCreator.GET_ALL_LIBRARIES, url, data, "POST", "", "JSON").bind(this);
```

Comment:
Does it make sense to still call the same `GET_ALL_LIBRARIES` action when attempting a search?